neqo-transport/src/recovery.rs

```diff
@@ -412,7 +412,7 @@ impl LossRecoverySpace {
             .sent_packets
             .iter_mut()
             // BTreeMap iterates in order of ascending PN
-            .take_while(|(&k, _)| Some(k) < largest_acked)
+            .take_while(|(&k, _)| largest_acked.is_none() || Some(k) < largest_acked)
```

Comment:
This might be easier to grok.

```suggestion
            .take_while(|(&k, _)| k < largest_acked.unwrap_or(PacketNumber::MAX))
```